### PR TITLE
variable: refine comments of variables

### LINKED ISSUE #32609: Code comments not in Golang code style
## Enhancement

A lot of code comments in tidb are not in line with Golang code style in `tidb_vars.go`.

image

### PR DESCRIPTION
**Pull request by jackysp**
Signed-off-by: Jack Yu <jackysp@gmail.com>

### What problem does this PR solve?

Issue Number: close #32609

Problem Summary: A lot of code comments in tidb are not in line with Golang code style in `tidb_vars.go`.

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [x] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

```release-note
None
```

### CODE REVIEW

File: sessionctx/variable/tidb_vars.go [hunk 1]
```diff
@@ -833,7 +836,7 @@ var (
 	ddlErrorCountlimit    int64 = DefTiDBDDLErrorCountLimit
 	ddlReorgRowFormat     int64 = DefTiDBRowFormatV2
 	maxDeltaSchemaCount   int64 = DefTiDBMaxDeltaSchemaCount
-	// Export for testing.
+	// MaxDDLReorgBatchSize exports for testing.
```
Comment (ekexium) [inline sessionctx/variable/tidb_vars.go:839]:
```suggestion
	// MaxDDLReorgBatchSize is exported for testing.
```

**Review by ekexium — APPROVED** [review]

**Review by tangenta — APPROVED** [review]

**Review by hawkingrei — APPROVED** [review]

**Comment by hawkingrei** [comment]
@XuHuaiyu @qw4990 PTAL

**Review by cfzjywxk — APPROVED** [review]

**Pull request merged**